libfwupdplugin/fu-udev-device.c

```diff
@@ -1403,7 +1403,11 @@ fu_udev_device_set_physical_id(FuUdevDevice *self, const gchar *subsystems, GErr
 		return FALSE;
 	}
 
-	if (g_strcmp0(subsystem, "pci") == 0) {
+	/* check for FDT first */
+	tmp = g_udev_device_get_property(udev_device, "OF_FULLNAME");
```

Comment:
is OF_FULLNAME just a I2C device thing?

Reply:
No. Its on DT systems for a lot of things. I have a whole udev output on

https://github.com/fwupd/fwupd/issues/7459

Reply:
So, if it;s not subsystem dependant, I think we should just set the physical ID from probe() if this property exists -- and then plugins don't need to call fu_udev_device_set_physical_id() at all.

Reply:
Sounds good